sites/all/themes/wille/sass/base/_vars.scss

```diff
@@ -99,3 +98,4 @@ $z-index-menu: 2;
 
 // Side margins on content on mobile.
 $mobile-margin: 20px;
+$border-radius: 20px;
```

Comment:
Jeg har lavet en rounded-box mixin, for der er forskellig border-radius i desktop og mobile.